launcher/ui/pages/instance/ManagedPackPage.cpp

```diff
@@ -146,6 +147,18 @@ bool ManagedPackPage::runUpdateTask(InstanceTask* task)
     return task->wasSuccessful();
 }
 
+bool ManagedPackPage::confirmUpdate()
```

Comment:
is this really needed? When the user is in the managed pack page they clearly already know the instance name and version they're updating to. Besides, this button is small, without shortcuts (meaning hard to press by accident), and there's normally a pretty sizeable time frame where the user can just abort the update and keep their instance intact. I feel like this extra confirmation would just get in the way of the user most of the time, instead of helping them :c